modules/python3-pycrypto.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "python3-pycrypto",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"pycrypto\" --no-build-isolation"
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "path": "/home/ben/Downloads/pycrypto-2.6.1.tar.xz",
```

Comment:
It's a local path.

Reply:
Well that explains why it builds for me :/